nixpkgs_review/nix/evalAttrs.nix

```diff
@@ -3,18 +3,35 @@
 with builtins;
 let
   pkgs = import <nixpkgs> { config = { checkMeta = true; allowUnfree = true; inherit allowAliases; }; };
-  lib = pkgs.lib;
+  inherit (pkgs) lib;
 
   attrs = fromJSON (readFile attr-json);
-  getProperties = name: let
-    attrPath = lib.splitString "." name;
-    pkg = lib.attrByPath attrPath null pkgs;
-    maybePath = builtins.tryEval "${pkg}";
-  in rec {
-    exists = lib.hasAttrByPath attrPath pkgs;
-    broken = !exists || !maybePath.success;
-    path = if !broken then maybePath.value else null;
-    drvPath = if !broken then pkg.drvPath else null;
-  };
+  getProperties = name:
+    let
+      attrPath = lib.splitString "." name;
+      pkg = lib.attrByPath attrPath null pkgs;
+      debugAttr = attr: lib.concatStringsSep "   " (lib.mapAttrsToList
```

Comment:
What is the goal of function i.e. what cases are you trying to debug?

Could it be replaced by `lib.generators.toPretty { allowPrettyValues = true; }`?